nbextensions/bulkImportCell/testAppObj-ci.json

```diff
@@ -1,898 +1,920 @@
 {
-  "app": {
-    "gitCommitHash": "25e7a896377f5ec50bd15a27ade9f279cb16cd0b",
-    "id": "kb_uploadmethods/import_fastq_sra_as_reads_from_staging",
-    "spec": {
-      "behavior": {
-        "kb_service_input_mapping": [
+  "app":{
```

Comment:
I switched out the jobs in this to use IDs from a local narrative with jobs in the appropriate states, making it easier to check and test. Unfortunately the JSON formatter I used has removed a whole load of spaces -- I didn't spot that until just now!